lib/client.ts

```diff
@@ -165,17 +165,17 @@ export default class Client {
       .then(res => res.richmenus);
   }
 
-  public setDefaultRichMenu(richMenuId: string): Promise<any> {
+  public setDefaultRichMenu(richMenuId: string): Promise<{}> {
     return this.post(URL.defaultRichMenu(richMenuId));
   }
 
-  public getDefaultRichMenuId(): Promise<string> {
+  public getDefaultRichMenuId(): Promise<Types.RichMenuResponse> {
```

Comment:
Oh, this change is wrong. I meant it's good to have `getDefaultRichMenu` other than `getDefaultRichMenuId`. The return type `string` was correct here.

Reply:
oh, I see.